### DISCUSSION TITLE
disable 7z

### DISCUSSION THREAD

**Discussion by adamantida** [MikeMaximus/gbm#314] [Ideas]
I would like to request that the files be saved in a different format other than 7z. Instead, I would prefer them to be stored as a folder.
Regarding the advantages of this approach, we can mention the quick ability to manipulate and access the saved files. Additionally, this method allows for easy compression into a custom archive format and the ability to restore the original backup if needed.

**Comment by MikeMaximus** [comment]
Hi adamantida,

I would like GBM to have the option for different types of compression (rar, zip, etc) as well as a straight file copy.  

However this would require I rewrite a large portion of the application to be more modular as well as require a lot of UI work and retesting.  I won't say it'll never be done, but it's something likely to be pushed off until I fully rewrite the application.

Thanks for your suggestion. I'll be moving it to the discussions section if you have anything to add.
